Example/Sources/Extensions/AlertService.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AlertService.swift
+//  ChatExample
+//
+//  Created by Mohannad on 12/25/20.
+//  Copyright © 2020 MessageKit. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+
+class AlertService {
+
+    
+    static func showAlert(style: UIAlertController.Style, title: String?, message: String?, actions: [UIAlertAction] = [UIAlertAction(title: "Ok", style: .cancel, handler: nil)], completion: (() -> Swift.Void)? = nil) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: style)
+        for action in actions {
+            alert.addAction(action)
+        }
+//        if let topVC = UIApplication.getTopMostViewController() {
+//            alert.popoverPresentationController?.sourceView = topVC.view
+//            alert.popoverPresentationController?.sourceRect = CGRect(x: topVC.view.bounds.midX, y: topVC.view.bounds.midY, width: 0, height: 0)
+//            alert.popoverPresentationController?.permittedArrowDirections = []
+//            topVC.present(alert, animated: true, completion: completion)
+//        }
+        
+        if let current = (UIApplication.shared.delegate as? AppDelegate)?.window?.rootViewController {
```

Comment:
Let's simply do `UIApplication.shared.delegate?.window??.rootViewController?.present(alert, animated: true, completion: completion)`? There is no need for the `AppDelegate` `as?`?